posydon/binary_evol/flow_chart.py

```diff
@@ -74,7 +75,7 @@
 # we infer the state, if core_definition_H_fraction=0.1 then surface_h1=0.1
 # and the state is H-rich_non_burning which we stil want to evolve thorugh
 # the step_CO_HeMS
-STAR_STATES_HE_RICH_EVOLVABLE.append('H-rich_non_burning')
+STAR_STATES_HE_RICH_EVOLVABLE.extend(['H-rich_non_burning', 'accreted_He_non_burning'])
```

Comment:
According to the comment, I'd not add `accreted_He_non_burning` here, because this shouldn't be a post-CE state and surface_h1=0.1 won't lead to `accreted_He`.
But maybe it is needed when using 1percent for the core boundary. Could you please recheck this?